contracts/test/MockSpokePool.sol

```diff
@@ -9,14 +9,24 @@ import "../SpokePoolInterface.sol";
  * @notice Implements abstract contract for testing.
  */
 contract MockSpokePool is SpokePoolInterface, SpokePool {
+    uint256 chainId_;
+
     constructor(
         address _crossDomainAdmin,
         address _hubPool,
         address _wethAddress,
-        address timerAddress
-    ) SpokePool(_crossDomainAdmin, _hubPool, _wethAddress, timerAddress) {}
+        address timerAddress,
+        uint256 _chainId
```

Comment:
As discussed IRL would be nice not to have to pass in a new constructor param. Instead default this to 0 and allow someone to override it